code/game/objects/items/weapons/mop.dm

```diff
@@ -28,19 +28,21 @@
 	GLOB.janitorial_equipment -= src
 	return ..()
 
-/obj/item/mop/proc/wet_mop(obj/o, mob/user)
-	if(o.reagents.total_volume < 1)
-		to_chat(user, "[o] is out of water!</span>")
-		if(istype(o, /obj/structure/mopbucket))
-			var/obj/structure/mopbucket/mopbucket = o
-			mopbucket.mopbucket_insert(user, o)
-		if(istype(o, /obj/structure/janitorialcart))
-			janicart_insert(user, o)
+/obj/item/mop/proc/wet_mop(obj/O, mob/user)
+	if(O.reagents.total_volume < 1)
+		to_chat(user, "[O] is empty!</span>")
```

Comment:
Needs a span/class tag or to remove the closing span at the end.

Reply:
```suggestion
		to_chat(user, "<span class='notice'>[O] is empty!</span>")
```